source/update.rst

```diff
@@ -37,6 +65,98 @@ The following steps will allow you to update both of them to current version
    $ kubectl create -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{{release}}}/deploy/operator.yaml
    $ kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
 
+
+
+.. _operator-update-smartupdates:
+
+Upgrading Percona Distribution for PostgreSQL
+---------------------------------------------
+
+Automatic upgrade
+*****************
+
+Starting from version 1.1.0, the Operator does fully automatic upgrades to
+the newer versions of Percona PostgreSQL Cluster within the method named *Smart
+Updates*.
+
+.. note:: Smart Updates have technical preview status and are disabled by
```

Comment:
Not a smart update but Version Service has technical preview status. And we disabled it by default (in CR), so  operator can't get new images from VS but if user sets a new images via CR and apply it the cluster will be updated in a smart way :)